SignalMessaging/attachments/EditContactShareNameViewController.swift

```diff
@@ -42,9 +42,11 @@ class ContactNameFieldView: UIView {
         createContents()
     }
 
+    private let hMargin = CGFloat(16)
+
     func createContents() {
         let vMargin = CGFloat(10)
-        self.layoutMargins = UIEdgeInsets(top: vMargin, left: 0, bottom: vMargin, right: 0)
+        self.layoutMargins = UIEdgeInsets(top: vMargin, left: hMargin, bottom: vMargin, right: hMargin)
```

Comment:
Do we want anything other than the system default horizontal margin?

Can this just be:

```
self.layoutMargins.top = vMargin
self.layoutMargins.bottom = vMargin
```

And an aesthetic nit: if an explicit hMargin is needed for some reason, seems like it might as well live next to the vMargin definition.

Reply:
> Do we want anything other than the system default horizontal margin?

I'm trying to match the design - and I think Myle's design looks better than the default margins.

Reply:
> if an explicit hMargin is needed for some reason, seems like it might as well live next to the vMargin definition.

Done!